drivers/ethernet/eth_nxp_enet_qos/eth_nxp_enet_qos_mac.c

```diff
@@ -0,0 +1,606 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT nxp_enet_qos_mac
+
+#include <zephyr/logging/log.h>
+#define LOG_LEVEL CONFIG_ETHERNET_LOG_LEVEL
+LOG_MODULE_REGISTER(eth_nxp_enet_qos_mac);
```

Comment:
This certainly works, but if you do not need `LOG_LEVEL` in the code, then this could be written like this too
```
LOG_MODULE_REGISTER(eth_nxp_enet_qos_mac, CONFIG_ETHERNET_LOG_LEVEL);
```